### PR TITLE
Patches from FreeBSD ports - Recast booleans as bool for clarity.

### PR DESCRIPTION
**Pull request by rozhuk-im**
More patches.
Bool conversions - for php update to 7x
"Clear on suspend" - was in FreeBSD ports

### CODE REVIEW

**Comment by h6w** [comment]
Hey.  Thanks for the PR.   Can you please include a link to where you got the patches, please?  Cheers.

**Comment by rozhuk-im** [comment]
https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=223154 - this is mine, not in ports now
https://svnweb.freebsd.org/ports/head/www/phpvirtualbox/files/patch-panes_mediumEncryptionPasswords.html?revision=446746&view=markup&sortby=file

https://svnweb.freebsd.org/ports/head/www/phpvirtualbox/files/?sortby=file

**Comment by rozhuk-im** [comment]
Should I remove 8d4e71a ? (It in https://github.com/phpvirtualbox/phpvirtualbox/pull/57 )

**Comment by h6w** [comment]
Yeah.  Can you split https://github.com/phpvirtualbox/phpvirtualbox/commit/8d4e71a730340c6b63262b2192e5323a86ab829d  into a separate PR?  I feel it's muddying the PR request a little and is really a feature request we need to consider separately.

Thanks!

**Comment by rozhuk-im** [comment]
https://github.com/phpvirtualbox/phpvirtualbox/commit/8d4e71a730340c6b63262b2192e5323a86ab829d is at https://github.com/phpvirtualbox/phpvirtualbox/pull/57
Anything else?

File: endpoints/lib/vboxServiceWrappers.php [hunk 1]
```diff
@@ -7881,6 +7881,10 @@ public function resize($arg_logicalSize)
 
     public function reset()
     {
+        // No password ID in case of decryption and de-/encryption are both handled here.
```
Comment (h6w) [inline endpoints/lib/vboxServiceWrappers.php]: Sorry to be a pedant, but I'm curious about these 4 lines.  They aren't simply recasting like the rest of the patch.  These lines already exist in changeEncryption and you're not blanking the vars like I'd expect in reset().  Can you explain this further?

Comment (rozhuk-im) [reply endpoints/lib/vboxServiceWrappers.php]: It is from https://github.com/phpvirtualbox/phpvirtualbox/pull/55/commits/ab7059433b7c7f7232d3e33435f02e920a9a2702
I merge together my changes with changes from other authors.
I will remove it.

**Comment by h6w** [comment]
Awesome.  Thanks!

File: endpoints/lib/vboxServiceWrappers.php (hunk 1 above)
Comment (h6w) [reply endpoints/lib/vboxServiceWrappers.php]: Yeah, that patch is patching changeEncryption(), which we have already done.  Whereas your patch add these lines to reset().  See above.

**Pull request merged**